File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> sol;
        
        for(int j=0;j<nums.size();j++){
            int comp=target-nums[j];
            for(int i=0;i<nums.size();i++){
                if(nums[i]==comp && i!=j && sol.size()<2){
                    sol.push_back(j);
                    sol.push_back(i);
                }
            }
        }
        return sol;
    }
};

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i = 0; i < nums.length ; i++){
            int b = target - nums[i];
            for(int j = i+1 ; j < nums.length ; j++){
                if(i!= j && nums[j] == b){
                int result[] = {i,j};
                return result;}

                else
                continue;
            }
        }
        return new int[]{};
    }
}